File: Practice/21/Python/21.py
def bmi(weight: float, height: float) -> float:
    return weight / (height*height)
    
def print_bmi(bmi: float) -> float:
    if   bmi < 18.5:
        print("Underweight")
    elif bmi < 25.0:
        print("Normal weight")
    elif bmi < 30.0:
        print("Overweight")
    else:
        print("Obesity")

weight, height = map(float, input("Введите вес и рост(в сантиметрах): ").split(" "))
print(weight, height)
print_bmi(bmi(weight, height*0.01))

File: Practice/21/С++/21/21/21.cpp
#include <iostream>
using namespace std;

double BMI(double weight, double height)
{

	double BMI = weight / (height * height);
	return BMI;

}
void printBMI(double BMI)
{
	string result;
	if (BMI < 18.5 && BMI > 0)
		result = "Underweight";
	else if (BMI >= 18.5 && BMI < 25)
		result = "Normal";
	else if (BMI >= 25 && BMI < 30)
		result = "Overweight";
	else if (30 <= BMI)
		result = "Obesity";
	else cout << "Error, incorrect index.";
	cout << result << endl;


}
int main()
{
	setlocale(LC_ALL, "Russian");
	double weight, height;
	cout << "Введите свой вес 'в килограммах' и рост в 'сантиметрах': ";
	cin >> weight >> height;
	printBMI(BMI(weight, height*0.01));
}